src/glmfit.jl

```diff
@@ -296,6 +296,19 @@ function StatsBase.fit!(m::AbstractGLM, y; wts=nothing, offset=nothing, dofit::B
     end
 end
 
+"""
+    fit(AbstractGLM, X, y, d, [l = canonicallink(d)]; <keyword arguments>)
```

Comment:
I don't think `AbstractGLM` is right: it should be `::Type{<:AbstractGLM}`. But better show the actual concrete type that users can pass, i.e. `GeneralizedLinearModel`.

Can you also give the type and meaning of positional arguments (see `?lm`)?

Reply:
Done